tests/run/nbd-tester-client.c

```diff
@@ -1743,6 +1772,8 @@ int main(int argc, char **argv)
 		case 'f':
 			testflags |= TEST_FLUSH;
 			break;
+		case 'I':
+			goto inetd_mode;
```

Comment:
This is not at all guaranteed to work.

some getopt() implementations parse the whole command line first, and/or rearrange options so that non-option arguments end up at the very end of the command line. Doing a goto to skip the rest of the command line is not a good idea in that context.

I would prefer it if instead the -I option would set a flag in the `testflags` thing (or flip a boolean somewhere else), and that the command be specified after a double-dash separator; this is documented to be "the end of options", and therefore does not have these issues.

(besides, this kind of use of goto makes things look a bit hairy)